src/App.js

```diff
@@ -158,10 +188,45 @@ function updateSettings(e) {
 let chord, notes;
 let svg;
 let triadType;
+let inversionType;
 
 function getCount() {
   return null;
 }
+
+function getInversion(chord, inv) {
+
+  const voices = chord.simple().length;
+  inv = inv % voices;
+  if (inv === 0) return chord;
+
+  let voicing = chord.voicing();
+  let i = 0;
+
+  if (voices - inv >= voices / 2) { // Move bottom notes up
+    while (i < inv) {
+      voicing[i] = voicing[i].add(Teoria.interval('P8'));
+      i++;
+    }
+    voicing = util.rotateElements(voicing, inv * -1);
+  } else { // Move top notes down
+    while (i < voices - inv) {
+      voicing[voices - (i + 1)] = voicing[voices - (i + 1)].add(Teoria.interval('P-8'));
+      i++;
+    }
+    voicing = util.rotateElements(voicing, (voices - inv));
+  }
+
+  let simple_inversion = [];
+  voicing.forEach(interval => {
+    simple_inversion.push(interval.toString());
+  });
+
+  chord.voicing(simple_inversion);
+
+  return chord;
+}
+
```

Comment:
Re-spells our inverted voicing as intervals. 
For example: 1st inversion chord goes from 
```
['P1', 'M3', 'M5']
``` 
to 
```
['M3', 'M5', 'P8']
```